.pre-commit-config.yaml

```diff
@@ -1,18 +0,0 @@
-files: '(screenpy_playwright|tests)/.*'
-fail_fast: false
-repos:
-  - repo: https://github.com/psf/black
-    rev: 24.4.2
-    hooks:
-      - id: black
-        language_version: python3.12
-  - repo: https://github.com/astral-sh/ruff-pre-commit
-    # Ruff version.
-    rev: v0.5.4
-    hooks:
-      - id: ruff
-  - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.11.0
-    hooks:
-      - id: mypy
-        language_version: python3.12
```

Comment:
Since we are removing the pre-commit, we should update the readme and remove the step that says to run `pre-commit install`